main/lsp/LSPFileUpdates.h

```diff
@@ -25,6 +25,18 @@ class LSPFileUpdates final {
 
     TypecheckingPath typecheckingPath = TypecheckingPath::Slow;
 
+    // The names of all the changed files involved in a fast path edit, and the index of their data in the
+    // `updatedFiles` vector. This vector stores paths because file refs can change between deciding if it's possible to
+    // take the fast path and actually taking it.
+    std::vector<std::pair<std::string, size_t>> fastPathChangedFiles;
```

Comment:
This doesn't seem necessary? Isn't this always the same as what's in `this->updatedFiles`?

Reply:
Great call! I was so caught up in piping the results from `fastPathFilesToTypecheck` out that I missed that 😅